code/game/objects/items/toys.dm

```diff
@@ -561,6 +561,8 @@
 		D.icon = 'icons/obj/chemical.dmi'
 		D.icon_state = "chempuff"
 		D.create_reagents(5)
+		reagents.log_bad_reagents(user, src)
+		user.investigation_log(I_CHEMS, "sprayed 1u from \a [src] \ref[src] containing [reagents.get_reagent_ids(1)] towards [A] ([A.x], [A.y], [A.z]).")
```

Comment:
Pretty much no reason other than `spray.dm` was already doing it.  If passing the ref doesn't really serve any purpose, I'll remove it.